src/tink/core/Future.hx

```diff
@@ -313,16 +329,7 @@ enum FutureStatus<T> {
 private interface FutureObject<T> {
 
   function getStatus():FutureStatus<T>;
-  /**
-   *  Registers a callback to handle the future result.
-   *  If the result is already available, the callback will be invoked immediately.
-   *  @return A `CallbackLink` instance that can be used to cancel the callback, no effect if the callback is already invoked
-   */
   function handle(callback:Callback<T>):CallbackLink;
-  /**
-   *  Makes this future eager.
-   *  Futures are lazy by default, i.e. it does not try to fetch the result until someone `handle` it
-   */
   function eager():Future<T>;
```

Comment:
I wonder if it makes sense for this method to return `Future<T>`, since all the implementations just `return this`, so maybe we could have `eager():Void` in `FutureObject` and then just `return this` in the abstract method?

Reply:
Sounds good ;)